cobc/ChangeLog

```diff
@@ -1,4 +1,20 @@
 
+2023-08-17  Fabrice Le Fessant <fabrice.le_fessant@ocamlpro.com>
+
+	* cobc.c: add new flags to output dependencies following gcc: -M to
+	  output deps only, -MD to output deps while compiling (in .d files),
+	  -MP to output phony targets, -MG to keep missing copybooks,
+	  -MQ <target> to Makefile-quote target
+
+2023-08-16  Fabrice Le Fessant <fabrice.le_fessant@ocamlpro.com>
+
+	* cobc.c, flags.def: add new flags to output dependencies:
+	  -foneline-deps outputs all dependencies on a single line instead of
+	  multiple lines; -fcopybook-deps outputs only copybook names instead
+	  of file paths. -fcopybook-deps also forces -E, -foneline-deps,
+	  -MT=copybooks, disables errors on missing copybooks and removes
+	  output on stdout.
+
```

Comment:
if you rewrote the changes please updated the dates, if you made significant adjustments then add a single "new" entry with the new date on top, otherwise it is fine to leave dates as is, but in any case: the entries should be located in the file according to their dates.